code/modules/hivemind/mobs.dm

```diff
@@ -302,7 +308,7 @@
 /mob/living/simple_animal/hostile/hivemind/bomber/death()
 	..()
 	gibs(loc, null, /obj/effect/gibspawner/robot)
-	explosion(get_turf(src), 0, 0, 2)
+	explosion(get_turf(src), 0, 1, 3) //explosion equal to a full welding fuel tank, deadly
```

Comment:
A walking fuel tank is very strong, I would probably nerf it's speed even more.

Reply:
We surely don't need a one shot explosive mobs for hive.

Reply:
Considering that you CAN NOT run into hive unknowingly, due to obvious wires, it's not that bad idea. If you are going to fight hive, you should be prepared. Melee is not a choice. But current explosions are already deadly enough for me, I was one shot exploded by it once.

Reply:
There are enough buffs in this PR, no need to make it this hard anyway.

Reply:
![image](https://user-images.githubusercontent.com/59490776/89080918-7def3e00-d38a-11ea-8bad-d583a51f88ff.png)
this is how much damage the bomber (with my shitcode) does.

2.5 speed seems to be enough to be able to run away from it, but fast enough that you can't run straight past it like you can with 2 speed. So gonna change the speed to that

Reply:
bomber now has 2.5 (slow enough to run away, fast enough to get to you if you try running through it) speed and 0,0,3 explosion (100brute dmg if you stand right in the middle of the explosion)